src/test/java/it/com/atlassian/bitbucket/jenkins/internal/util/BitbucketUtils.java

```diff
@@ -80,6 +80,40 @@ public static void createBranch(String project,
                 .getBody();
     }
 
+    public static void createSshPublicKey() {
+        Map<String, Object> createSshKeyRequest = new HashMap<>();
+        createSshKeyRequest.put("text", TestUtils.readFileToString("/ssh/test-key.pub"));
+
+        RestAssured.given()
+                .queryParam("user", BITBUCKET_ADMIN_USERNAME)
+                .log()
+                .ifValidationFails()
+                .auth()
+                .preemptive()
+                .basic(BITBUCKET_ADMIN_USERNAME, BITBUCKET_ADMIN_PASSWORD)
+                .contentType(ContentType.JSON)
+                .body(createSshKeyRequest)
+                .expect()
+                .statusCode(201)
+                .when()
+                .post(BITBUCKET_BASE_URL + "/rest/ssh/1.0/keys");
+    }
+
+    public static void deleteSshPublicKey() {
```

Comment:
Both the creation and teardown are being done in individual jenkins instances at the behest of the test, not the bbs jenkins rule- but that means getting the ID to the test to tell it which ssh key to destroy is kind of pushing extra work to the test I think should be handled by the rule instead. This is a clumsier solution, just wiping all keys for good measure, and while it shouldn't cause issues on a remote instance, if people have ssh keys on their admin accounts locally, this will wipe them. If reviewers prefer having tests storing keys (or restructuring so all tests have SSH credentials and the keys are handled by the rule), please let me know- it's fairly trivial to fix.